### ISSUE TITLE
Make lkm fails

### ISSUE THREAD

**Issue by epixinvites** [darlinghq/darling#943]
```
/home/skynet/github/darling/src/external/lkm/osfmk/duct/duct_kern_thread_act.c: In function ‘thread_get_state_internal’:
/home/skynet/github/darling/src/external/lkm/osfmk/duct/duct_kern_thread_act.c:208:54: error: ‘TIF_IA32’ undeclared (first use in this function)
  208 |    if (!test_ti_thread_flag(task_thread_info(ltask), TIF_IA32))
      |                                                      ^~~~~~~~
/home/skynet/github/darling/src/external/lkm/osfmk/duct/duct_kern_thread_act.c:208:54: note: each undeclared identifier is reported only once for each function it appears in
/home/skynet/github/darling/src/external/lkm/osfmk/duct/duct_kern_thread_act.c: In function ‘thread_set_state’:
/home/skynet/github/darling/src/external/lkm/osfmk/duct/duct_kern_thread_act.c:495:55: error: ‘TIF_IA32’ undeclared (first use in this function)
  495 |     if (!test_ti_thread_flag(task_thread_info(ltask), TIF_IA32))
      |                                                       ^~~~~~~~
make[6]: *** [scripts/Makefile.build:279: /home/skynet/github/darling/src/external/lkm/osfmk/duct/duct_kern_thread_act.o] Error 1
make[5]: *** [Makefile:1800: /home/skynet/github/darling/src/external/lkm] Error 2
make[5]: Leaving directory '/usr/lib/modules/5.11.2-arch1-1/build'
make[4]: *** [Makefile:615: default] Error 2
make[4]: Leaving directory '/home/skynet/github/darling/src/external/lkm'
make[3]: *** [src/external/lkm/CMakeFiles/lkm.dir/build.make:77: src/external/lkm/CMakeFiles/lkm] Error 2
make[2]: *** [CMakeFiles/Makefile2:56341: src/external/lkm/CMakeFiles/lkm.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:56348: src/external/lkm/CMakeFiles/lkm.dir/rule] Error 2
make: *** [Makefile:18422: lkm] Error 2
```

I cloned the repository at approximately UTC 2021/03/05 1500, and I followed all the instructions from https://docs.darlinghq.org/build-instructions.html. Installed everything, but still failed.

clang version 11.1.0
cmake version 3.19.6
5.11.2-arch1-1

**Comment by facekapow** [comment]
Duplicate of #933.
